lib/assessment.rb

```diff
@@ -5,35 +5,54 @@
 # Create a class called Dog. It should have, as attributes, name, breed, and age.
 # Make all of those attributes read-only.
 
-##
-# your code here
-##
+class Dog
+  def initialize (name, breed, age)
+    @name = name
+    @breed = breed
+    @age = age
+  end
+    attr_reader :name
+    attr_reader :breed
+    attr_reader :age
+end
+
 
 # Question 2
 # Create a new Dog object.
 
-Answer.dog = nil # replace nil with your code
+Answer.dog = ("dog", "doberman", 4)
```

Comment:
Not quite right. You've written a list here. How do you create a new instance of a `Dog`?